geopandas/io/tests/test_sql.py

```diff
@@ -282,6 +282,34 @@ def test_from_postgis_custom_geom_col(self, connection_postgis, df_nybb):
 
         validate_boro_df(df, case_sensitive=False)
 
+    def test_from_postgis_multiple_geom_col(self, connection_postgis, df_nybb):
+        con = connection_postgis
+        geom_col = "the_geom"
+        geom_col_2 = "the_geom_2"
+        create_postgis(con, df_nybb, geom_col=geom_col)
+
+        # create second column with geometry
+        try:
+            cursor = con.cursor()
+            sql = """ALTER TABLE IF EXISTS public.nybb
+                ADD COLUMN {0} geometry;
+                UPDATE public.nybb
+                SET {0}={1};""".format(
+                geom_col_2, geom_col
+            )
+            cursor.execute(sql)
+        finally:
+            cursor.close()
+            con.commit()
+
+        sql = "SELECT * FROM nybb;"
+        df = GeoDataFrame.from_postgis(sql, con, geom_col=[geom_col, geom_col_2])
+
+        validate_boro_df(df, case_sensitive=False)
+
+        # Make sure also the second column was properly loaded as MultiPolygon
+        assert pd.Series(df[geom_col_2].type).dropna().eq("MultiPolygon").all()
```

Comment:
```suggestion
        assert df[geom_col_2].type.eq("MultiPolygon").all()
        assert df.crs.equals(df[geom_col_2).crs)
```

This should be enough. `.type` is already Series and there should be no missing geoms.

plus I would check if the CRS is correctly assigned.

Reply:
Thanks for the comments! I just did a quick test and it seemed it the crs was still empty. I will investigate later today.